content/report/mapping.md

```diff
@@ -1,10 +1,12 @@
 ---
+doc-note-type: draft
 layout: standalone_resource
 title: "Mapping To Rule"
 permalink: /standards-guidelines/act/report/mapping/
 ref: /standards-guidelines/act/report/mapping/
 lang: en
 type_of_guidance: false
+sidebar: false
```

Comment:
Do we not want to show this in the sidebar?

Reply:
Nope, that's the page with the empty side bar :) This gets rid of it.